scripts/arch/immutability.py

```
@@ -112,6 +113,139 @@ def _split(text: str) -> tuple[dict, str]:
+        # остаться в списке: обещание расширяется, а не подменяется.
+        if (
+            current.is_file()
+            and _declaration_in(current.read_text(encoding="utf-8"), old_declaration)
+            and old_reference not in new_names
+        ):
+            return False
+        pool = {reference.partition("::")[0] for reference in new_names}
+        covered = _covered_names(repo, base_ref, old_reference, pool)
+        if not covered:
+            return False
+        # Покрытие — включение, а не равенство: список вправе называть больше,
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `а` (CYRILLIC SMALL LETTER A). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF003)